static/gsApp/views/seerAutomation/repos.tsx

```diff
@@ -1,9 +1,9 @@
 import SeerSettingsPageWrapper from 'getsentry/views/seerAutomation/components/seerSettingsPageWrapper';
 
-export default function SeerAutomationRepos() {
+export default function SeerAutomationSettings() {
```

Comment:
shouldn't this be repos?

```suggestion
export default function SeerAutomationRepos() {
```